requirements.txt

```diff
@@ -0,0 +1,131 @@
+absl-py==1.3.0
```

Comment:
General advice is to keep here only project-specific requirements, e.g. packages that you are importing.
In this way, someone can easily see what technologies you use in the repo and install them manually. Also, your environment may contain packages that are already unrelated to the app (move to other libs), dev packages (black, mypy), or hardware-specific (cudnn-c11).